samples/encryption.js

```diff
@@ -63,10 +63,14 @@ function uploadEncryptedFile(bucketName, srcFilename, destFilename, key) {
   // const srcFilename = 'Local file to upload, e.g. ./local/path/to/file.txt';
   // const destFilename = 'Remote destination for file, e.g. file_encrypted.txt';
 
+  // See the "Generating your own encryption key" section above.
+  // const key = 'A customer-supplied key';
```

Comment:
How about: A base64 encoded customer-supplied key? That way know what the sample code is expecting without reading the code.

Reply:
The "Generating your own encryption key" is from the docs, which is where these samples are read from: https://cloud.google.com/storage/docs/using-encryption-keys#storage-generate-encryption-key-nodejs

I think it's a good idea to add more context by saying "A base64 encoded customer-supplied key", but I think it would help to leave the reference to the "Generating your own encryption key" section, which can elaborate.

Reply:
I popped in some more "base64" references.